cmd/lk/egress.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 LiveKit, Inc.
+// Copyright 2024 LiveKit, Inc.
```

Comment:
nit: if the intent was to indicate year of modification, I would recommend using a range instead.. i.e. `2023-2024`.

changing it to the current year would lose information about when the file was originally created

Reply:
Fair point! I did a little git scripting and got these updated to reflect the full history of each file.